bento/crates/api/src/helpers.rs

```diff
@@ -10,18 +10,33 @@ use workflow_common::{
     AUX_WORK_TYPE, COPROC_WORK_TYPE, EXEC_WORK_TYPE, ExecutorResp, JOIN_WORK_TYPE, PROVE_WORK_TYPE,
 };
 
+/// Extract reserved value from api_key string
+/// Format: "reserved:N" where N is a positive number (e.g., "reserved:5" or "reserved:10")
+/// Returns 0 if no reserved value is specified
+pub fn extract_reserved(api_key: &str) -> i32 {
```

Comment:
Not too familiar with convention in Bento, but it seems `api-key == user-id`? If so, lets just use one term in each code base, so rename to user id here?

Reply:
Felt more correct to make it referenced as API key because the user id semantics might be a subset of the API key (currently a bit unused). Happy to switch though